### PR TITLE
feat: add on demand loading of 360 images into a visualization object

### PR DESCRIPTION
**Pull request by christjt**
~~THIS PR DEPENDS ON #2494 BEING MERGED FIRST~~

# Description

Adds a method on the 360 image entity that initializes and loads the 360 image from the files API and applies it to a mesh which is added to the scene. Currently there is no management around this (f.ex. that you can only have one 360 image visible at a time, etc.) as this will be implemented later.

# Checklist:

Here is a checklist that should completed before merging this given feature. 
Any shortcomings from the items below should be explained and detailed within the contents of this PR.

- [x] I am proud of this feature.
- [x] I have performed a self-review of my own code.
- [ ] I have manually tested this for different scenarios (different models, formats, devices, browsers).
I only have access to one data set from DVMP at the moment, but will test more as I get more access.
- [x] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
Will handle when more of the API is fleshed out
- [x] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [x] I have refactored the code for readability to the best of my ability.
- [x] I have checked that my changes do not introduce regressions in the public documentation.
- [x] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [x] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [x] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
Will handle when more of the API is fleshed out
- [x] I have noted down and am currently tracking any technical debt introduced in this PR.
- [x] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: viewer/packages/360-images/src/Image360Entity.ts [hunk 1]
```diff
@@ -50,10 +92,10 @@ export class Image360Entity {
     drawOuterCircle();
     drawHoverSelector();
 
-    const spriteMaterial = new THREE.SpriteMaterial({ map: new THREE.CanvasTexture(canvas), depthTest: false });
+    const spriteMaterial = new THREE.SpriteMaterial({ map: new THREE.CanvasTexture(canvas), depthTest: true });
```
Comment (pramodcog) [inline viewer/packages/360-images/src/Image360Entity.ts:95]: `depthTest` by default is `true` for materials

File: viewer/packages/data-providers/src/types.ts [hunk 2]
```diff
@@ -29,6 +29,7 @@ export type Image360Descriptor = {
 
 export type Image360Face = {
   face: 'front' | 'back' | 'left' | 'right' | 'top' | 'bottom';
+  mimeType: 'image/jpeg' | 'image/png';
```
Comment (pramodcog) [inline viewer/packages/data-providers/src/types.ts:32]: `jpeg` & `png` are the only format we support?

Comment (christjt) [reply viewer/packages/data-providers/src/types.ts:32]: For now, yes.

**Review by haakonflatval-cognite — APPROVED** [review]
LGTM, just a shame I don't have access to any means to test it :P

File: viewer/packages/360-images/src/Image360Entity.ts [hunk 3]
```diff
@@ -50,10 +92,10 @@ export class Image360Entity {
...
     const sprite = new THREE.Sprite(spriteMaterial);
     sprite.updateMatrixWorld();
-    sprite.renderOrder = 3;
+    sprite.renderOrder = 4;
```
[the remark this answers is not in the document]
Comment (christjt) [reply viewer/packages/360-images/src/Image360Entity.ts:98]: Other way around. It will show.

**Review by pramodcog — APPROVED** [review]
LGTM!

**Pull request merged**